requirements.txt

```diff
@@ -11,7 +11,7 @@ Markdown==2.4.1
 Pygments==1.6
 WTForms==1.0.4
 beautifulsoup4==4.3.2
-celery==3.1.17
+celery==3.1.18
```

Comment:
Ah, this was a compatibility issue with my mobile machine running LXLE (Ubuntu 14.04 LTS downgrade). The easy fix was to bump up the version number. This can be removed, if you'd like.